keyboards/dz60/keymaps/crd/keymap.c

```diff
@@ -0,0 +1,50 @@
+#include QMK_KEYBOARD_H
+
+#define ______ KC_TRNS
```

Comment:
QMK defines a keycode to do this automatically.

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/quantum/quantum_keycodes.h#L29-L31

Please delete Line 3, and replace all instances of  
`______` (six underscores) with
`_______` (seven underscores).